core/src/wallet/common/synchronizers/AbstractBlockchainExplorerAccountSynchronizer.h

```diff
@@ -272,9 +272,9 @@ namespace ledger {
                                             std::shared_ptr<SynchronizationBuddy> buddy) {
 
                 buddy->logger->info("SYNC BATCHES");
-                //For ethereum like wallets, one account corresponds to one ETH address,
+                //For ethereum and ripple like wallets, one account corresponds to one ETH address,
```

Comment:
[…] to one ETH or XRP address, respectively

Reply:
Why not 👍